### PR TITLE
[FIX] Encode avatar url to prevent CSS injection

### PR DESCRIPTION
**Pull request by rodrigok**
@RocketChat/core

### CODE REVIEW

File: packages/rocketchat-ui-message/client/message.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 			{{else}}
 				<button class="thumb user-card-message" data-username="{{u.username}}" tabindex="1">
 					<div class="avatar">
-						<div class="avatar-image" style="background-image:url({{avatar}});"></div>
```
Comment (engelgabriel) [inline packages/rocketchat-ui-message/client/message.html]: Should we save the URL encoded on the DB?

Comment (karlprieb) [inline packages/rocketchat-ui-message/client/message.html]: I think we can do this in the future, because we will need a migration to do that

Comment (rodrigok) [inline packages/rocketchat-ui-message/client/message.html]: @engelgabriel I don't know now, cuz we save usernames there too, can be a problem

Comment (gronke) [inline packages/rocketchat-ui-message/client/message.html]: We could do it in the client. The server should reject malformed messages with a proper error code.

**Pull request merged**